### PR TITLE
feat(properties): document how to configure Bonita with properties

### PR DESCRIPTION
**Pull request by educhastenier**
* with Setup Tool
* via Java Sysprops
* via Env vars

### CODE REVIEW

File: modules/runtime/pages/configuring-bonita-with-properties.adoc [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+      - BONITA_RUNTIME_PROPERTIES_ORDER_LEGACY_MODE=true
+# ...
+----
+
+
+== limitations
+
+Some properties cannot be changed via System Properties or Environment Variables:
+
+* TODO
```
Comment (vhemery) [inline modules/runtime/pages/configuring-bonita-with-properties.adoc]: No TODO in the final doc site... 😉

**Review by vhemery — CHANGES_REQUESTED** [review]
I can't find the configuring-bonita-with-properties.adoc from the navigation menu.
It should be added in runtime-configuration-index and in runtime-nav.adoc

Also see suggestions inline.

**Comment by vhemery** [comment]
> I can't find the configuring-bonita-with-properties.adoc from the navigation menu. It should be added in runtime-configuration-index and in runtime-nav.adoc
> 
> Also see suggestions inline.

Argh.. my most meaningfull comment was lost...
I was suggesting to use alternative presentation for:

- In a Linux Shell:
- When starting a Bonita Docker container:
- In a docker-compose.yml file:
See https://github.com/bonitasoft/bonita-doc/edit/feat/properties/document_bonita_properties/modules/runtime/pages/bonita-docker-installation.adoc and search for `[tabs]` for examples.

File: modules/runtime/pages/configuring-bonita-with-properties.adoc (hunk 1 above)
Comment (educhastenier) [reply modules/runtime/pages/configuring-bonita-with-properties.adoc]: Fixed in next commit

**Comment by educhastenier** [comment]
> > I can't find the configuring-bonita-with-properties.adoc from the navigation menu. It should be added in runtime-configuration-index and in runtime-nav.adoc
> > Also see suggestions inline.
> 
> Argh.. my most meaningfull comment was lost... I was suggesting to use alternative presentation for:
> 
> * In a Linux Shell:
> * When starting a Bonita Docker container:
> * In a docker-compose.yml file:
>   See https://github.com/bonitasoft/bonita-doc/edit/feat/properties/document_bonita_properties/modules/runtime/pages/bonita-docker-installation.adoc and search for `[tabs]` for examples.

Feedbacks integrated.
Thx for the suggestions.

**Review by vhemery — APPROVED** [review]
Looks perfect to me

**Pull request merged**